src/routes/+layout.svelte

```diff
@@ -106,8 +106,42 @@
             }
         }
     }
+
+    onMount(() => {
+        const reoIdentity = {
+            username: $user.email,
+            type: $user.password ? 'email' : 'github',
+            firstname: $user.name.split(' ')[0],
+            lastname: $user.name.split(' ')[1]
+        };
+
+        if (dev || !Object.keys($user).length) return;
+        // @ts-expect-error
+        Reo.identify(reoIdentity);
```

Comment:
Typically, you would identify a user right after they successfully log in and/or you get the user.